jujugui/static/gui/src/app/components/user-profile/entity/test-entity.js

```diff
@@ -321,7 +333,7 @@ describe('UserProfileEntity', () => {
     });
   });
 
-  it('can toggle to the expanded view', () => {
+  xit('can toggle to the expanded view', () => {
```

Comment:
It looks like we're having the same issue with the deployment bar notification tests: https://github.com/juju/juju-gui/blob/develop/jujugui/static/gui/src/app/components/deployment-bar-notification/test-deployment-bar-notification.js